zerver/openapi/zulip.yaml

```diff
@@ -10274,6 +10285,14 @@ paths:
               any server version. Clients can then use the `zulip_feature_level` in the
               `/register` response or the presence/absence of a `user_settings` key to
               determine where to look for the data.
+
+            - `linkifier_url_template`: Boolean for whether the client accepts linkifiers
+              that uses [RFC 6570](https://www.rfc-editor.org/rfc/rfc6570.html) compliant URL
+              template for linkification. If false, the client will get empty lists for event
+              types `realm_linkifiers`.
```

Comment:
It's an empty list for `realm_linkifiers` in the register response, and no `realm_linkifiers` events at all, right?

Reply:
I'm looking for a better way to phrase this. Currently using "If false, an empty list will be sent in the `/register` response and `realm_linkifiers` events will be dropped".

Reply:
That seems pretty good. I'd just add an explicit mention of which property in the `/register` response you're talking about.

Oh and as written, it sounds like the default is effectively *true*.

So combining those, one version would be:

> If false or unset, then the `realm_linkifiers` list in the `/register` response will be empty if present, and no `realm_linkifiers` events will be sent.

Reply:
OK, that's updated.